apis/infix/atrium-api-infix/src/commonMain/kotlin/ch/tutteli/atrium/api/infix/en_GB/mapLikeToContainInOrderOnlyCreators.kt

```diff
@@ -183,8 +191,6 @@ infix fun <K, V, T : MapLike> EntryPointStep<K, V, T, InOrderOnlySearchBehaviour
  *   or it does not have elements (is empty).
  *
  * @since 0.18.0
- *
- * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.MapLikeToContainInOrderOnlyCreatorSamples.entriesOf
```

Comment:
Accidentally added this to the wrong function last PR😨

Reply:
no worries, didn't saw it in the review either, good you caught it now :+1: